UnicornRender/video/source/Graphics.cpp

```diff
@@ -39,19 +38,19 @@ bool Graphics::Init(const DriverType& driver)
 
     m_driver = driver;
 
-    LOG_INFO("Graphics initialization started.");
+    spdlog::get(utility::Logger::LoggerName())->info("Graphics initialization started.");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.